html/cross-origin-embedder-policy/header-parsing.https.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html>
+  <meta charset="utf-8">
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+<body>
+<script>
+'use strict';
+function createIframe(t, values) {
+  const parent = document.createElement('iframe');
+  const child = document.createElement('iframe');
+  const params = new URLSearchParams(values.map((value) => ['value', value]));
+  parent.setAttribute('src', `resources/empty-coep.py?${params}`);
+  document.body.appendChild(parent);
+  t.add_cleanup(() => parent.remove());
+
+  return new Promise((resolve, reject) => {
+      parent.onload = resolve;
+      parent.onerror = reject;
+    })
+    .then(() => {
+        child.setAttribute('src', '/common/blank.html');
+        parent.contentDocument.body.appendChild(child);
+        return new Promise((resolve) => child.onload = resolve);
+      })
+    .then(() => child);
+}
+
+[
+  [],
+  [''],
+  ['jibberish'],
+  ['require\u2014corp'], // non-ASCII character (em dash)
```

Comment:
This test has a lot of layers of abstraction, in particular whatever URLSearchParams does to this, and whatever the Python software does to this... I wonder if there's any way to make it clearer exactly what bytes the header will contain?

Reply:
`.asis` files may be the most literal mechanism, but that will introduce a different kind of indirection--lots of small files. I'm happy to refactor, but I'll wait until someone confirms that's desirable.

Reply:
Could you at least add a comment as to what bytes this comes out to be in practice? As is my best guess is that it ends up being the bytes for the modulo sign and some numbers.

Reply:
> my best guess is that it ends up being the bytes for the modulo sign and some numbers

The character is transmitted to wptserve via the query string as `%E2%80%94`. The server deserializes the value and inserts the character U+2014 into the header value.

> Could you at least add a comment as to what bytes this comes out to be in practice?

That's the intent of the existing comment, but I've only been thinking in terms of the response header. By "comes out", do you mean that the comment should describe the value in the query string parameter? Or would you like to see a complete list of all the bytes (that is, including the ASCII characters)? Or am I just way off?

Reply:
Well, em dash is not a byte; a byte is a number in the range 0-255. And does UTF-8 get involved (since Python apparently takes strings and then converts them to bytes)? That could insert a leading byte or two... I'm hoping to see what bytes this ends up delivering.

Reply:
I checked with PacketPeeper. The bytes the server sends are E2 80 94.

Reply:
Let's clarify that the server will emit UTF-8 bytes. That does seem like it would limit what you can test though. E.g., you wouldn't be able to emit 0xFF.

Reply:
I've changed the test to more directly test 0xFF in 0001299